docscan/sandbox/request/task/supplementary_document_text_data_extraction_task_test.go

```diff
@@ -0,0 +1,106 @@
+package task
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/getyoti/yoti-go-sdk/v3/docscan/sandbox/request/filter"
+)
+
+func ExampleSupplementaryDocumentTextDataExtractionTaskBuilder() {
+	docFilter, err := filter.NewDocumentFilterBuilder().Build()
+	if err != nil {
+		fmt.Printf("error: %s", err.Error())
+		return
+	}
+
+	task, err := NewSupplementaryDocumentTextDataExtractionTaskBuilder().
+		WithDocumentFilter(docFilter).
+		WithDocumentField("some-key", "some-value").
+		WithDocumentField("some-other-key", map[string]string{
+			"some-nested-key": "some-nested-value",
+		}).
+		Build()
+	if err != nil {
+		fmt.Printf("error: %s", err.Error())
+		return
+	}
+
+	data, _ := json.Marshal(task)
```

Comment:
We have a ticket in the backlog to check these errors, even though we'd definitely find the error from the JSON output in the line below, we're wanting to show good practice by always checking err values. Don't mind if you want to change this here, or we can do them all in one go when we do that ticket.

Reply:
(Applies to other tests in this PR too)

Reply:
I've fixed the examples related to this PR in eca0353

Will open a separate PR to fix in unrelated examples